README.md

```diff
@@ -1,7 +1,14 @@
 ## Introduction
-Facebook launched their searchable archive of U.S. political advertisements on May 24, 2018. We devise a data collection methodology to obtain a large set of 267,000 political ads from Facebook’s political ad archive. We find that in total, ads with political content have generated at least 1,435,089,000 impressions and have cost their sponsors $13,913,300 and possibly up to 3,884,705,000 impressions and spent $71,754,827 on advertising with U.S. political content. Individual Facebook pages that sponsor political content ads, on average, generate at least 72,501 total impressions and spend at least $703 on advertising. Facebook has also launched an API for programmatic access to this archive.
+
```

Comment:
No need to change anything for this comment, it's just an FYI.
It's our fault for having a large paragraph in one line and not having the empty line as markdown formatters expect, but it's good to try and avoid changing lines outside of what you were actually working on because when a PR moves unrelated text, it's hard to understand exactly what changed.

Reply:
Sure sure, that was my auto-formatter being a little overzealous. I'll update my PR.